OneCommunityGlobal/HighestGoodNetworkApp
# Akshit taking over for Cheng-Yun Members.jsx Unit Test

# Description
Implementation unit test fuctions for Members.jsx

## Related PRS (if any):
None

## Main changes explained:

- Create Members.test.jsx file for testing Members.jsx components 

## How to test:
1. check into current branch
2. Make sure that the development server is running in the background.
3. do `npm install` and `npm test src/components/Projects/Members/Members.test.jsx` to run this PR locally

## Screenshots or videos of changes:
<img width="620" alt="unit test cases fix" src="https://github.com/user-attachments/assets/e7b97ac2-6375-4514-a91d-643bb2e9373f">
